src/components/displayComponents/previousEducation.js

```diff
@@ -13,13 +13,22 @@ const PreviousEducation = props => {
           {/*  */}
           <List.Content>
             <div>
-              {props.item.degree} - {graduationOptionsMap[props.item.graduation]} in {props.item.field}
-              <p styleName="style.gray">
-                {props.item.institute + ", "}
+              {props.item.degree} -{' '}
+              {graduationOptionsMap[props.item.graduation]} in{' '}
+              {props.item.field}
+              <div styleName="style.gray">
+                {props.item.institute + ', '}
```

Comment:
Use template literals

Reply:
This was original implementation of SHP, should I be changing this?